packages/schematics/angular/package.json

```diff
@@ -9,6 +9,7 @@
   ],
   "schematics": "./collection.json",
   "dependencies": {
+    "@angular-devkit/architect": "0.0.0",
```

Comment:
While it does make sense to add Architect here, we've tried not to do it because the schematics shouldn't strictly need it. It's a bit murky really... but it would be better to not add the dependency at the moment and wait until we figure out the intended deps.

Reply:
Makes sense, removed the dep!